cfgov/teachers_digital_platform/models/activity_index_page.py

```diff
@@ -393,13 +393,13 @@ def validate_results_per_page(request):
     A utility for parsing the requested number of results per page.
 
     This should catch an invalid number of results and always return
-    a valid number of results, defaulting to 5.
+    a valid number of results, defaulting to 25.
     """
     raw_results = request.GET.get('results')
     if raw_results in ['10', '25', '50']:
```

Comment:
If it defaults to 25, we could probably remove 25 here. But I don't think it's required to merge this PR.